mscore/musescore.cpp

```diff
@@ -3673,6 +3673,7 @@ static bool doConvert(Score *cs, const QString& fn)
       else if (fn.endsWith(".metajson"))
             return mscore->saveMetadataJSON(cs, fn);
       // unknown file type
+      fprintf(stderr, "Unknown file type!\n");
```

Comment:
Is this the only place where `stderr` is used?

Reply:
no. A couple lines later too

Reply:
In converter mode, which is on the command line, stderr is the right place to go for error messages IMHO